vendor/k8s.io/kubernetes/pkg/kubectl/cmd/delete.go

```diff
@@ -284,7 +284,7 @@ func ReapResult(r *resource.Result, f cmdutil.Factory, out io.Writer, isDefaultD
 	return nil
 }
 
-func DeleteResult(r *resource.Result, out io.Writer, ignoreNotFound bool, shortOutput bool, mapper meta.RESTMapper) error {
+func DeleteResult(r *resource.Result, out io.Writer, ignoreNotFound bool, gracePeriod int, shortOutput bool, mapper meta.RESTMapper) error {
```

Comment:
as in upstream, a godoc would be nice about this int. time.Duration would be obvious, int is not.